ravenml/data/write_dataset.py

```diff
@@ -75,14 +84,25 @@ def load_image_ids(self):
         """
         raise NotImplementedError
 
-    def interactive_filter(self):
-        """Method assumes that 'image_ids' and 'tags_df' have already been found and allows
+    def set_size_filter(self, set_sizes: dict=None):
+        """Method assumes that 'image_ids' has already been found and allows
             user to filter through them for subsets they choose to use based on tags
```

Comment:
your PR description indicates that this method doesn't require tags to function (and thus doesn't need metadata files, only image ids). Is this docstring wrong or is my understanding wrong?